### ISSUE TITLE
About generating 802.1AR certificates

### ISSUE THREAD

**Issue by nidiascampos** [CiscoDevNet/MUD-Manager#88]
Hi,
I've got errors while generating 802.1AR certificates. The scripts in examples/certs_and_keys  do not generate the 8021ARintermediate/private/8021ARintermediate.key.

Please see the errors in the
[errors.txt](https://github.com/CiscoDevNet/MUD-Manager/files/5748097/errors.txt)
 attached file.

Can someone help me?

**Comment by elear** [comment]
Hiya!  Ca you let me know what platform you are running on and in particular what version of openssl you are using?

Thanks!

**Comment by nidiascampos** [comment]
Hi.
I'm using Ubuntu 20.04.1 LTS and OpenSSL 1.1.1f.

**Comment by elear** [comment]
Thanks.  I'll check into it.  Please have a bit of patience due to the holiday.

**Comment by elear** [comment]
Ok, there are definitely some problems with this script.  Working on it.

**Comment by elear** [comment]
@nidiascampos thanks for your report.  These are some contrib scripts that weren't quite doing the job.  I've cleaned them up but need to do some regression testing before I check them in.  Probably another day or three.

**Comment by elear** [comment]
Created PR #89 that should cover this and other several other bugs.  This PR is from the dev branch. Can you pull the dev branch and let me know if you have any issues?

**Comment by nidiascampos** [comment]
Hi, 
I've run the new code and other errors have appeared.
[errors2.txt](https://github.com/CiscoDevNet/MUD-Manager/files/5764587/errors2.txt)

**Comment by elear** [comment]
Yes, I am now seeing this as well.  My apologies.  A fix is forthcoming.

**Comment by elear** [comment]
This should now be corrected.  My apologies again.  N.B. I think these scripts are too chatty.  If you agree, I can clean that up.
